README.md

```diff
@@ -346,73 +255,64 @@ The following platforms are currently supported:
 
 To add a new platform definition, add a new class to `krun/platform.py`.
 
-## Development and Debug Switches
-
-If you are making changes to Krun itself (for example, to add a new platform or
-virtual machine definition), there are a few switches which can make your life
-easier.
 
-  * `--debug=<level>`: Sets the verbosity of Krun.  Valid debug levels are:
-     `DEBUG`, `INFO`, `WARN`, `DEBUG`, `CRITICAL` and `ERROR`. The default is
-     `WARN`. For real benchmarks you should use the default.
+## Testing your configurations
 
-  * `--quick`: There are several places where Krun would normally wait using
-    sleeps or a polling loop. These are essential for real benchmarking, but
-    annoying for development. Use `--quick` to skip these delays.
+Before doing a full run of an experiment, you should perform a quick(ish) test
+of your configuration. This can be achieved with:
 
-  * `--no-user-change`: Without this flag, For each process execution, Krun
-    will use a fresh user account called 'krun'. This involves deleting any
-    existing user account (with `userdel -r`) and creating a new user account
-    (with `useradd -m`).  This switch disables the use of a fresh user account,
-    meaning that `userdel` and `useradd` are not invoked, nor does Krun switch
-    user; the user Krun was invoked with is used for benchmarking.
+```sh
+$ /path/to/krun/krun.py --dry-run --quick --debug=INFO config.krun
+```
 
-  * `--dry-run`: Fakes actual benchmark processes, making them finish
-    instantaneously.
+\laurie{i think this should have `--no-reboots` on too?}
```

Comment:
The logic is the other way around. We have `--hardware-reboots` to turn on rebooting.